src/routes.js

```diff
@@ -5,12 +5,16 @@ import UsersPage from './components/users/UsersPage';
 import ServerStatusPage from './components/serverStatus/ServerStatusPage';
 import LoginPage from './components/login/LoginPage';
 import CreateUserPage from './components/users/CreateUserPage';
+import LogoutPage from './components/logout/LogoutPage';
+import CheckAuth from './api/redirectLoginApi';
```

Comment:
Descargue la branch y me tira este error en esta linea:
`/Users/bernabe/Proyects/ash_web/src/routes.js (1/0)
  ✖  9:8  No default export found in module  import/default `

A vos tambien te lo tira? si te lo tira arreglalo